manifests/a/Apple/iPodReset/1.0.4.71/Apple.iPodReset.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Apple.iPodReset
+PackageVersion: 1.0.4.71
+Installers:
+- InstallerUrl: https://download.info.apple.com/Mac_OS_X/061-4529.20080508.PdLde/iPodResetUtilitySetup.exe
+  Architecture: neutral

```

```suggestion
  Architecture: x86
  UnsupportedOSArchitectures:
    - x64
    - arm64
```
> Action start 9:11:08: LaunchConditions.
MSI (s) (7C:A8) [09:11:08:990]: Product: iPod Reset Utility -- This installer is intended for 32-bit versions of Windows.